### ISSUE TITLE
13.3 (55.l) Claim is not necessarily true.

### ISSUE THREAD

**Issue by beemirang** [pat-rogers/Ada-202x-WG9-Informal-Review#109]
"The programmer might think that My_Device'Size is 8, and that My_Device'Address points at an 8-bit location. However, this [is not]{might not be] true."

It might be also be true, depending on the hardware, implementation, etc

**Comment by stephen-baird-751** [comment]
I think the suggested "might not be true" wording change is a substantial improvement over the current wording (for the reasons given by the reviewer), but I would ever-so-slightly prefer  "is not guaranteed" over that.

**Comment by sttaft** [comment]
I agree there is an issue, and I also like Steve's suggested wording:

> The programmer might think that My_Device'Size is 8, and that My_Device'Address points at an 8-bit location. However, this is not {guaranteed to be} true.

**Comment by ARG-Editor** [comment]
OK. Added this to the AARM AI, AI12-0005-1.
